www/%team/edit/close.spt

```diff
@@ -0,0 +1,22 @@
+from aspen import Response
+from gratipay.utils import get_team
+
+[----------------------------------------------------------------------------]
+
+if user.ANON:
+    raise Response(401, _("You need to log in to access this page."))
+
+request.allow('POST')
+
+team = get_team(state)
+
+if not user.ADMIN and user.participant.username != team.owner:
+    raise Response(403, _("You are not authorized to access this page."))
+
+if team.is_closed:
+    raise Response(403, _("Already closed."))
```

Comment:
If you try to close again by URL for some reason, you would get this error. All my latest commit did was hide the link to the project from the project owner's profile.